src/main.rs

```diff
@@ -238,3 +256,125 @@ fn parse_args() -> Result<LndCfg, ArgsError> {
 
     Ok(LndCfg::new(address, cert_file, macaroon_file))
 }
+
+#[derive(Debug)]
+enum SetOnionBitError {
+    UnimplementedPeersService,
+    SetBitFail,
+    UpdateAnnouncementErr(Status),
+    GetInfoErr(Status),
+}
+
+impl Error for SetOnionBitError {}
+
+impl fmt::Display for SetOnionBitError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            SetOnionBitError::UnimplementedPeersService => write!(
+                f,
+                "Peer service is unimplemented. Remember to
```

Comment:
nit: let's go for lower case for `Display` + error strings, upper case for logs?

Going to add a `contributing.md` to spell this out, would be nice to at least attempt to be consistent in the beginning :')

Reply:
soudns good!